### DISCUSSION TITLE
How to Enable Contributor Role?

### DISCUSSION THREAD

**Discussion by xCJPECKOVERx** [photoprism/photoprism#3946] [Q&A] [answered]
I noticed in a part of the docs [here](https://www.photoprism.app/pro/kb/roles) there is a new contributor role listed for sharing albums and allowing uploads. However, there doesn't appear to be any way to create a user with this role.
Is the feature just not added yet (running preview build 231206-0935d9fab) but the docs are being updated, or is it rather only going to be available to "pro" business licences?

Any help or guidance for activating this would be appreciated, as this appears to be a role/feature many of us have been waiting a long time for (sharing albums with upload permissions).

**Comment by lastzero** [comment, marked as answer]
Please note that this overview only applies to [PhotoPrism® Pro](https://www.photoprism.app/teams#compare). Information on the user management features of [our personal editions](https://www.photoprism.app/editions#compare) is available in the user guide at https://docs.photoprism.app/user-guide/users/.

The Contributor role was introduced specifically for professional users such as newspaper agencies who want to let external photographers upload their pictures to a specific folder. It thus requires additional configuration parameters that are not available in our personal editions and is also not suitable for sharing specific albums with upload rights.

We are currently working on additional authentication options for all editions and will release them as soon as possible:

- https://github.com/photoprism/photoprism/issues/3943
- https://github.com/photoprism/photoprism/issues/808
- https://github.com/photoprism/photoprism/issues/782
- https://github.com/photoprism/photoprism/issues/98

This year so far:

- we developed a free, high-resolution vector world map that is now available to all users
- released the initial user management that you can use right now
- took care of a long list of pull requests that were waiting to get merged, e.g. the [Places cluster view](https://github.com/photoprism/photoprism/releases/tag/230923-e59851350) that many users had been waiting for
- provided installation packages for Linux as an alternative to our Docker images
- continued to improve our documentation for you
- and provided ongoing fixes for recently reported issues

**Reply by xCJPECKOVERx** [reply]
Ok sounds good, thanks for the quick reply! It looks like I misinterpreted the use case for contributors anyway.
Really appreciate everything the Photoprism team is doing 😊

Cheers
